web_console/dynamic-plugin/overview-dynamic-plugin.adoc

```diff
@@ -39,11 +39,15 @@ conster Header: React.FC = () => {
 ----
 
 * Avoid selectors that could affect markup outside of your plugins components, such as element selectors. These are not APIs and are subject to change. Using them might break your plugin. Avoid selectors like element selectors that could affect markup outside of your plugins components.
+* Provide valid JavaScript Multipurpose Inernet Mail Extension (MIME) type using the `Content-Type` response header for all assets served by your plugin web server. Each plugin deployment should include a web server that hosts the generated assets of the given plugin.
```

Comment:
Should "Multipurpose Inernet Mail Extension" be "Multipurpose Internet Mail Extension"?